docs/controls/AdaptiveGridView.md

```diff
@@ -10,23 +9,100 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, AdaptiveGridView,
 
 The **AdaptiveGridView Control** presents items in a evenly-spaced set of columns to fill the total available display space. It reacts to changes in the layout as well as the content so it can adapt to different form factors automatically.
 
-If there are not enough items to fill one row, the control will stretch the items until all available space is filled. This can result in much wider items than specified. If you prefer your items to always stay close to the DesiredWidth, you can set the **StretchContentForSingleRow** property to **false**, to prevent further stretching.
+There are 3 ways to use this Control. 
+
+* You can set `DesiredWidth` and `ItemHeight`, which will scale the **width** of each item, and adjust the number of columns on demand, using all horizontal space.
+
+* To maintain aspect ratio, You can set `DesiredWidth` only. This will allow the **height** and **width** of each item, and adjust the number of columns on demand, using all horizontal space.
```

Comment:
Not setting the ItemHeight has nothing to do with maintaining aspect ratio. If you do not set the ItemHeight, then the height of the rows will be whatever the item height is

Reply:
Technically this is related to maintaining aspect ratio, because the ItemHeight will change when resizing.

Reply:
It's not related to maintaining aspect ratio. Yes, it is required to maintain aspect ratio. Not setting the item height just means that rows will take up as much space as required. 

Example: Take the sample app. Remove ItemHeight setter and change the Image in the template to be `<Image Source="{Binding Thumbnail}" Stretch="None"/>`